content/resources/service-delivery-project-reflection.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: service-delivery-project-reflection
+date: 2024-12-03 12:00:00 -0400
+title: "Service Delivery Project Reflection"
+summary: "Use this template, shared at the 2024 Spring Community Summit, to improve your own service delivery."
+
+# Controls how this page appears across the site
+# 0 -- hidden
+# 1 -- visible
+# 2 -- highlighted
+weight: 0
```

Comment:
I suggested weight = 0 so that people with the link can access, but anyone looking at a topic page or using the search wouldn't encounter.  It's confusing resource to encounter out of context.